src/display.s

```diff
@@ -0,0 +1,42 @@
+display:
+; ////// UPDATE SPRITES POSITION \\\\\\
+; // Isaac \\
+		ld a, (global_.isaac.y)
+		ld b,a
+		ld a, (global_.isaac.x)
+		ld c,a
+//top left
```

Comment:
Is this right ? shouldn't comment lines be prefixed with ';' ?

Reply:
wla-dx supports ASM style comments (with ';') as well as C style comment (with '//' and '/*   */').